confgenerator/testdata/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Golden error messages expect inputs to have Unix line endings.
+input.yaml text eol=lf
```

Comment:
Our tests depended on the default Travis behavior of using LF line endings even on Windows. This makes the dependency explicit.